doc/sphinx/addendum/generalized-rewriting.rst

```diff
@@ -861,7 +861,7 @@ are applied using the tactic :n:`rewrite_strat @rewstrategy`.
 :n:`@rewstrategy ; @rewstrategy`
    composition
 
-:n:`choice {+ @rewstrategy }`
+:n:`choice {+ {+; @rewstrategy } }`
```

Comment:
This seems wrong, we need parentheses now for sequencing with choice

Reply:
It is indeed wrong. It needed an update. Fixed.